tmt/steps/report/reportportal.py

```diff
@@ -0,0 +1,89 @@
+import dataclasses
+import io
+import os
+import zipfile
+from typing import List, Optional
+
+import click
+import junit_xml
+
+import tmt.steps.report
+import tmt.utils
+
+from . import junit
+
+
+@dataclasses.dataclass
+class ReportPortalData(tmt.steps.StepData):
```

Comment:
Recently a base class for "report" plugins has been added, `tmt.steps.report.ReportStepData`, please, use it instead of `tmt.steps.StepData`.